Content.Shared/Whitelist/EntityWhitelistSystem.cs

```diff
@@ -60,6 +60,98 @@ public bool IsValid(EntityWhitelist list, EntityUid uid)
 
         return list.RequireAll;
     }
+    /// The following are a list of "helper functions" that are basically the same as each other
+    /// to help make code that uses EntityWhitelist a bit more readable because at the moment
+    /// it is quite clunky having to write out component.Whitelist == null ? true : _whitelist.IsValid(component.Whitelist, uid)
+    /// several times in a row and makes comparisons easier to read
+
+    /// <summary>
+    /// Helper function to determine if Whitelist is not null and entity is on list
+    /// </summary>
+    public bool IsWhitelistPass(EntityWhitelist? whitelist, EntityUid uid)
+    {
+        if (whitelist == null)
+            return false;
+
+        return IsValid(whitelist, uid) == true;
+    }
+
+    /// <summary>
+    /// Helper function to determine if Whitelist is not null and entity is not on the list
+    /// </summary>
+    public bool IsWhitelistFail(EntityWhitelist? whitelist, EntityUid uid)
+    {
+        if (whitelist == null)
+            return false;
+
+        return IsValid(whitelist, uid) == false;
+    }
```

Comment:
```suggestion
    public bool IsWhitelistPass(EntityWhitelist? whitelist, EntityUid uid)
    {
        if (whitelist == null)
            return false;

        return IsValid(whitelist, uid);
    }

    /// <summary>
    /// Helper function to determine if Whitelist is not null and entity is not on the list
    /// </summary>
    public bool IsWhitelistFail(EntityWhitelist? whitelist, EntityUid uid)
    {
        if (whitelist == null)
            return false;

        return !IsValid(whitelist, uid);
    }
```
Since the null checks have been handled at this point, I'm pretty sure these should be able to be simplified. (The rest of the helpers too)

Reply:
yeah ok